javascripts/govuk/selection-buttons.js

```diff
@@ -83,7 +83,9 @@
   }
   SelectionButtons.prototype.getClickHandler = function () {
     return function (e) {
-      this.markSelected($(e.target))
+      var target = $(e.target)
+      this.markSelected(target)
+      target.trigger('focus')
```

Comment:
Might be worth to add a comment here linking to this PR so that people don't remove this in the future when they test in Chrome and go "Hey this already happens, this line is not necessary."